src/keri/db/basing.py

```diff
@@ -939,7 +939,10 @@ def __init__(self, headDirPath=None, reopen=False, **kwa):
         self._kevers.db = self  # assign db for read through cache of kevers
 
         if (mapSize := os.getenv(KERIBaserMapSizeKey)) is not None:
-            self.MapSize = int(mapSize)
+            try:
+                self.MapSize = int(mapSize)
+            except ValueError:
+                logger.exception("KERI_BASER_MAP_SIZE must be an integer value!  Will continue to use default keripy value of 104857600!")
```

Comment:
I do not believe this is an appropriate change.  It is not a bug to fail fast when an illegal value is entered for MapSize.  It prevents an application from running with a different value and failing some time later (perhaps much later) when a database map file blows up.  Trying to track that down would be very difficult.

If you want a better error message you can rethrow a new exception with a clearer message or just leave the code as is.

Reply:
Sure.  I'll rethrow with a clearer message.

Reply:
@pfeairheller fixed